frontend/src/router.js

```diff
@@ -31,13 +33,13 @@ export default new Router({
       props: ({query: { cid }}) => ({
```

Comment:
is this dying tho? `cid` that is ?

Reply:
Yes it's dying; out of scope of this issue though